bigquery_etl/cli/query.py

```diff
@@ -1591,7 +1591,7 @@ def _update_query_schema(
         click.echo(f"{query_file} dry runs are skipped. Cannot update schemas.")
         return
 
-    tmp_tables = copy.copy(tmp_tables)
+    tmp_tables = copy.deepcopy(tmp_tables)
```

Comment:
This should fix issues where schema update fails for `baseline_clients_last_seen` with something like `invalid project ID 'moz-fx-data-shared-prod.tmp.moz-fx-data-shared-prod.tmp.moz-fx-data-shared-prod'.`